modules/test/playwright/tests/portal-workflow-kaleo-designer-web/timerNotification.spec.ts

```diff
@@ -0,0 +1,112 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {workflowPagesTest} from '../../fixtures/workflowPagesTest';
+import {getRandomInt} from '../../utils/getRandomInt';
+
+export const test = mergeTests(apiHelpersTest, loginTest(), workflowPagesTest);
+
+const timerNotifications = [
+	{
+		notificationDescription: 'notificationDescription0' + getRandomInt(),
+		notificationName: 'notificationName0' + getRandomInt(),
+		notificationTypeEmail: true,
+		notificationTypeUser: true,
+		recipientType: 'role',
+		recipientTypeData: {
+			roleName: 'Account Manager',
+		},
+		template: 'template0' + getRandomInt(),
+		templateLanguage: 'freemarker',
+	},
+	{
+		notificationDescription: 'notificationDescription1' + getRandomInt(),
+		notificationName: 'notificationName1' + getRandomInt(),
+		notificationTypeEmail: true,
+		notificationTypeUser: true,
+		recipientType: 'scriptedRecipient',
+		recipientTypeData: {
+			script: 'script' + getRandomInt(),
+			scriptLanguage: 'groovy',
+		},
+		template: 'template1' + getRandomInt(),
+		templateLanguage: 'text',
+	},
+] as Notification[];
+
+test('LPD-16281 can create timer notifications', async ({
+	apiHelpers,
+	diagramViewPage,
+	nodePropertiesSidebarPage,
+	page,
+	processBuilderPage,
+	sourceViewPage,
+	timerPage,
+}) => {
+	const singleApproverWorkflowDefinition =
+		await apiHelpers.headlessAdminWorkflow.getWorkflowDefinitionByName(
+			'Single Approver'
+		);
+
+	const postWorkflowDefinition =
```

Comment:
This variable should have another name, like `copyOfSingleApprover`, or only `workflowDefinition`. `postWorkflowDefinition` sounds like it is pointing to a function.